setup.py

```diff
@@ -15,6 +15,9 @@
     ),
     package_dir={"": "src"},
     packages=[""],
+    install_requires=[
+        "typing-extensions>=4.0,<5"
```

Comment:
Install it only when python is lower then 3.10

Reply:
3.11, done